packages/jest-snapshot/src/__tests__/utils-test.js

```diff
@@ -30,3 +36,17 @@ test('getSnapshotPath()', () => {
     path.join('/abc', 'cde', '__snapshots__', 'a-test.js.snap'),
   );
 });
+
+test('saveSnapshotFile()', () => {
+  const data = {
+    myKey: '<div>\r\n</div>', 
+  };
+
+  const file = path.join(__dirname, '.tmp.snapshot');
+  saveSnapshotFile(data, file);
+
+  const content = fs.readFileSync(file, 'utf-8');
```

Comment:
We also need to make sure in `beforeEach` that this file is deleted and doesn't exist from the beginning of the test run.